taqueria-plugin-flextesa/README.md

```diff
@@ -1,20 +1,37 @@
-# Taqueria Flextesa Plugin
+---
+title: Flextesa Plugin
+---
+
+[Flextesa](https://tezos.gitlab.io/flextesa/) is a flexible Tezos sandbox environment that's easy to work with. It runs a Tezos node and network in a Docker container on your local machine
 
 This plugin provides the ability to easily configure and run Flextesa sandbox instances within Taqueria that you can use for development and testing
+ 
+## Overview
+
+With the Flextesa plugin you configure various sandboxes for use in Taqueria that you can start, stop, and query from the CLI or VS Code command palette
 
-## Tasks
 
-The Flextesa plugin provides the following tasks to Taqueria:
-- `start sandbox`
-- `stop sandbox`
-- `list accounts`
+Some helpful things to know:
+- Multiple sandboxes can be configured and run concurrently
+- You can configure the Tezos `protocol` for each sandbox to test against current, and future network upgrades
+- Accounts and balances are pre-configured per-sandbox and do not persist their state
+    - Accounts and balances will be re-initialized each time they are started
+- Sandboxes can be added to Taqueria environments and then targeted from the CLI using the `--env [envName]` option 
 
 ## Requirements
 
-- Taqueria v0.0.6 or later
 - Node.js v16 or later
 - Docker v0.8.4 or later
 
+## Build Instructions
+
+To build the plugin locally from source, open the `taqueria-plugin-flextesa` directory and run the following:
```

Comment:
Rather than go to the `taqueria-plugin-flextesa` directory, users should run the `npm run build...` calls from the root.